CrosswordMe.py

```diff
@@ -1,46 +1,98 @@
 from random import randint
+from random import sample
 
-#input
+def print_puzzle(puzzle):
+    #Prints the puzzle puzzle on a board
+    for line in range(line_count):
+        for position in range(column_count):
+            print(str(puzzle[line][position]) + " ", end="") #Prints character in current position
+        print() #Prints Newline
 
-words = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"]
-user_input = ""
-print(len(words))
+def word_input():
+    #Collects the words for the puzzle
+    words = ["YEAHAA"] * 5
+    #, "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"]
+    user_input = ""
+    """while 1 == 1:
+        user_input = input("Word: ")
+        if user_input.lower() == "done":
+            break
+        words.append(user_input)"""
+    return words
 
-"""while 1 == 1:
-    user_input = input("Word: ")
-    if user_input.lower() == "done":
-        break
-    words.append(user_input)"""
+def define_field_size():
```

Comment:
I've talked to Lasse about globals the other day. He said, that global variables are a bad idea, but global constants (which thoose are) are fine. What do you think?